### ISSUE TITLE
Escaped quotes not working in CSV

### ISSUE THREAD

**Issue by AbdealiJK** [yandex/ClickHouse#2357]
I have a CSV with the line:
```
1,"\"Walt\""
```
I am running:
cat data.csv | clickhouse-client --query="INSERT INTO persons FORMAT CSV";

This fails with the message:
```
Column 0, name: pid, type: Nullable(String), parsed text: "1"
Column 1, name: pname, type: Nullable(String), parsed text: "<DOUBLE QUOTE><BACKSLASH><DOUBLE QUOTE>"
ERROR: There is no delimiter (,). "W" found instead.
```

It looks like the escaped double quotes: `\"` does not work in clickhouse.
Any notes on how to make it work ?

**Comment by filimonov** [comment]
That's because standart defined only one type of escaping quotes - by doubling it.
Check p. 7 of https://tools.ietf.org/html/rfc4180

**Comment by AbdealiJK** [comment]
Hm, got it. Most common tools allow both (pandas, excel, mysql, etc.) - would be beneficial for a bunch of users to avoid doing an additional step.
My data was very large and if I tried doing that I would have gotten a Harddisk full error so I wan unable to do any good formatting - got sed to remove those for now though.

Would be very useful if \ support could be added - but I get that it may not be a priority as there are a good number of workarounds.
Closing
